### ISSUE TITLE
Datensatz vom 19.12.2023 ohne Falldaten vom Wochenende 

### ISSUE THREAD

**Issue by MaHi84** [robert-koch-institut/SARS-CoV-2-Infektionen_in_Deutschland#12]
Der in der Nacht auf Dienstag aktualisierte Datensatz enthält keine neue Falldaten (vom Wochenende). Jüngstes Meldedatum ist der 15.12.2023. Betrifft ebenfalls den Archiv-Datensatz.

**Comment by HannesWuensche** [comment]
Hallo @MaHi84,

vielen Dank für den Hinweis. Ich habe es an die Fach-Kolleg:innen weitergegeben und wir schauen uns den Fehler an. 
Leider haben wir derzeit einen hohen Krankenstand und es kann sein, dass sich die Lösung dadurch etwas verzögert. 

Beste Grüße
@HannesWuensche

**Comment by MaHi84** [comment]
Das Repository COVID-19_7-Tage-Inzidenz_in_Deutschland wurde heute Morgen mit den neuesten Falldaten aktualisiert. Soweit ich das richtig beobachtet habe, waren die Daten gestern ebenfalls von dem Problem betroffen.

Ggf. hilft es zwischenzeitig das Skript einmal manuell auszuführen?

**Comment by HannesWuensche** [comment]
Hallo @MaHi84 ,

die Daten waren seit dem 16.12. betroffen. 
Wir konnten den Fehler identifizieren und mit dem Datensatz heute morgen beglichen. 

Oder haben wir etwas übersehen?

Beste Grüße
@HannesWuensche

**Comment by MaHi84** [comment]
Richtig, der Datensatz COVID-19_7-Tage-Inzidenz_in_Deutschland scheint aktuell zu sein.

Die Tabelle Aktuell_Deutschland_SarsCov2_Infektionen.csv wurde aber noch nicht aktualisiert. Gleiches gilt für die Archivdaten und zenodo.

**Comment by HannesWuensche** [comment]
Liebe Alle, 

wir konnten das Problem lösen und haben die Daten aktualisieren. 

Beste Grüße
@HannesWuensche 
für das Team RKI | Open Data

**Comment by Rubber1Duck** [comment]
Die Archiv Datei vom 19.12. wird nicht aktualisiert?
